app/views/user/details.phtml

```diff
@@ -68,11 +70,16 @@
                 <button type="submit" class="btn btn-important" name="action" value="update"><?= _t('gen.action.update') ?></button>
                 <button type="submit" class="btn btn-attention confirm" name="action" value="purge"><?= _t('gen.action.purge') ?></button>
                 <button type="submit" class="btn btn-attention confirm" name="action" value="delete"><?= _t('gen.action.remove') ?></button>
-                <?php if ($isAdmin && !$this->isDefaultUser): ?>
+                <?php if ($isAdmin && !$isDefault): ?>
                     <button type="submit" class="btn btn-attention confirm" name="action" value="demote"><?= _t('gen.action.demote') ?></button>
                 <?php elseif (!$isAdmin): ?>
                     <button type="submit" class="btn btn-attention confirm" name="action" value="promote"><?= _t('gen.action.promote') ?></button>
                 <?php endif; ?>
+                <?php if ($enabled && !$isDefault): ?>
+                    <button type="submit" class="btn btn-attention confirm" name="action" value="disable"><?= _t('gen.action.disable') ?></button>
```

Comment:
```suggestion
                    <button type="submit" class="btn btn-attention" name="action" value="disable"><?= _t('gen.action.disable') ?></button>
```

![image](https://user-images.githubusercontent.com/1008324/84587727-0b4df180-ae22-11ea-9854-e2c075fb6960.png)

This is a reversible action, so we might not need the confirmation dialogue box